app/src/main/java/edu/cascadia/mobile/apps/movies/EditMovie.java

```diff
@@ -23,28 +24,40 @@ protected void onCreate(Bundle savedInstanceState) {
         setSupportActionBar(toolbar);
 
         mDatabase = movieDatabase.getInstance(this);
+        final Intent EditIntent = this.getIntent();
+        int movieId = EditIntent.getIntExtra("movieIndex", 0);
```

Comment:
Make these values instance variable so that you can refer to them in your FAB button